app/build.gradle

```diff
@@ -30,6 +30,17 @@ android {
     }
 }
 
+configurations.all {
+    resolutionStrategy.eachDependency { details ->
+        def requested = details.requested
+        if (requested.group == "com.android.support") {
+            if (!requested.name.startsWith("multidex")) {
+                details.useVersion "26.+"
```

Comment:
The post you are referring to uses exact numbers when supplying version. So in our case, this should be 28.0.0 and nothing else.

Then again I don't really get how this change connects to the rest of the pull request.

Reply:
I copied and pasted that, so the "26.+" was not researched and if "28.0.0" makes sense, that can be changed.

As to how it relates to the pull request, it's that this bit of build definition prevents startup stack traces `OnUnhandledKeyEventListener`. 

So the pull request has two completely different "things", that each cause stack traces when the app starts.  This is just one of the two.

Reply:
Ok, only thing is that what I read this will override the settings you do in gradle.build and that would potentially give the implementation older versions.